src/topology/algebra/infinite_sum.lean

```diff
@@ -287,12 +306,11 @@ section tsum
 variables [add_comm_monoid α] [topological_space α] [t2_space α]
 variables {f g : β → α} {a a₁ a₂ : α}
 
-lemma has_sum_unique : has_sum f a₁ → has_sum f a₂ → a₁ = a₂ := tendsto_nhds_unique at_top_ne_bot
-
-lemma tsum_eq_has_sum (ha : has_sum f a) : (∑b, f b) = a := has_sum_unique (has_sum_tsum ⟨a, ha⟩) ha
+lemma tsum_eq_has_sum (ha : has_sum f a) : (∑b, f b) = a :=
+has_sum_unique (summable.has_sum ⟨a, ha⟩) ha
 
 lemma has_sum_iff_of_summable (h : summable f) : has_sum f a ↔ (∑b, f b) = a :=
```

Comment:
`summable.has_sum_iff`?

Reply:
This has prompted me to check the file again, and I have found two or three other lemmas that could benefit from the dot notation. Done.